src/webaudio.js

```diff
@@ -702,7 +702,11 @@ export default class WebAudio extends util.Observer {
         this.scheduledPause = null;
 
         this.startPosition += this.getPlayedTime();
-        this.source && this.source.stop(0);
+        try {
+            this.source && this.source.stop(0);
+        } catch (err) {
+            // no-op, see https://github.com/katspaugh/wavesurfer.js/issues/1473
```

Comment:
can you add a little more info so you only have to visit the ticket if you need all details